bin/main.ml

```diff
@@ -1,18 +1,56 @@
 open Diffcessible
 open Cmdliner
+open ExtUnix.Specific
+
+let create_empty_diff_content () =
+  "diff --git a/Error b/Error\n--- a/Error\n+++ b/Error\n"
+
+let setup_input = function
+  | Some path ->
+      if not (Sys.file_exists path) then failwith ("File not found: " ^ path);
+      let ic = In_channel.open_bin path in
+      (Some ic, Notty_unix.Term.create ())
+  | None ->
+      let tty_path = ttyname Unix.stdout in
+      let tty_fd = Unix.openfile tty_path [ Unix.O_RDWR ] 0o600 in
```

Comment:
`ttyy_fd` is never closed.

Reply:
Oops, forgot about that one

Reply:
You just added a `close` after the `Notty_unix.Term.t` is created but before the program could run. That's not right, the file descriptor should be open until the program closes.